Makefile

```diff
@@ -310,3 +304,10 @@ fuzzer-deps:
 run-cairo-compiled-fuzzer:
 	cd fuzzer
 	cargo +nightly fuzz run --fuzz-dir . cairo_compiled_programs_fuzzer
+
+wasm-demo:
+	cd examples/wasm-demo; \
+	cairo-compile src/array_sum.cairo --no_debug_info --output src/array_sum.json && \
+	wasm-pack build --target=web && \
+	npm install live-server && \
+	npx live-server
```

Comment:
These are new deps AFAICT. We should add them both to our deps targets/scripts and to the docs.

Reply:
These dependencies are only needed for the wasm-demo, which is not an integral part of the project, so I wouldn't force its dependencies for any user of the project (the main purpose of this PR is to separate it's build from the project's build). There is a separate README with instructions on how to build & run the demo, and its dependencies, so we could remove this target altogether